detection/evasion/empty_root_environ_linux.sql

```diff
@@ -97,4 +98,4 @@ WHERE
 GROUP BY
   p.pid
 HAVING
-  count == 0;
+  count = = 0;
```

Comment:
Should this go back to:
```suggestion
  count == 0;
```
?